rackcity/views/it_instance_views.py

```diff
@@ -134,12 +134,13 @@ def instance_add(request):
     if not serializer.is_valid(raise_exception=False):
         failure_message += str(serializer.errors)
```

Comment:
Wait does is_valid automatically call our new validator?? And then we just had to override save to make it also call the validator? If so that's big time

Reply:
YES is_valid does indeed call all validators on the model because of how we defined our serializers. save needs to be overridden for the cases where save is called when is_valid is NOT, because save does not call all validators on the model (seems stoopid to me but that's what overriding is for)